docs: add remote documentation (#331)
okteto/docs: src/content/reference/cli.mdx

| _--var_                                     |   (list)   | Set a pipeline variable (can be set more than once)                                                                                                            |
| _--wait_                                    |   (bool)   | Wait until the pipeline finishes (defaults to false)                                                                                                           |
| _--no-bash_                                 |   (bool)   | Execute the command using the container's default shell instead of bash (defaults to false)                                                                    |

> | _--remote_                                  |   (bool)   | Force run deploy commands in remote                                                                                                                            |

Reviewer: How about this?

```suggestion
| _--remote_                                  |   (bool)   | Run the deploy commands in your Okteto cluster
```

| _--volumes_                              |  (bool)  | Destroy the persistent volumes created by the development environment (defaults to `false`)                                                             |
| _--wait_                                 |  (bool)  | Wait until the development environment is destroyed (defaults to `false`)                                                                               |
| _--no-bash_                              |  (bool)  | Execute the command using the container's default shell instead of bash (defaults to false)                                                             |
| _--all_                                  |  (bool)  | Destroy everything (defaults to `false`)                                                                                                                |

> | _--remote_                               |  (bool)  | Force run deploy commands in remote                                                                                                                     |

Reviewer: ```suggestion
| _--remote_                               |  (bool)  | Run the destroy commands in your Okteto cluster                                                                                                                  |
```

> | _--remote_                               |  (bool)  | Run the deploy commands in your Okteto cluster                                                                                                          |

Reviewer: ```suggestion
| _--remote_                               |  (bool)  | Run the destroy commands in your Okteto cluster                                                                                                          |
```
